### PR TITLE
TECH-826 add structured candidates field to application model

### PR DESCRIPTION
**Pull request by ericbroder**

### CODE REVIEW

File: nominations/templates/admin/application_pdf.html [hunk 1]
```diff
@@ -99,12 +99,69 @@ <h2>
...
+
+      <section>
+
         <h2>
           State of the Race
         </h2>
 
         {{ app.state_of_the_race|safe }}
 
+      </section>
```
Comment (cjmabry) [inline nominations/templates/admin/application_pdf.html]: My understanding is that David is generating these manually right now so a feature flag here isn't required, but can we verify that is the case before we ship PDF changes for something vols won't be using yet.

File: nominations/views.py [hunk 2]
```diff
@@ -507,7 +507,10 @@ class ApplicationPDFView(PDFTemplateView):
 
     def get_context_data(self, **kwargs):
         app_id = self.request.GET.get('id')
-        app = get_object_or_404(Application, pk=app_id)
+        app = get_object_or_404(
+            Application.objects.select_related(),
```
Comment (cjmabry) [inline nominations/views.py]: Nice.

File: nominations/templates/admin/application_pdf.html (hunk 1 above)
Comment (ericbroder) [reply nominations/templates/admin/application_pdf.html]: I spoke to David about this, and he said only the admin UI field for volunteers needs to be gated, not pdf changes. Although volunteers have access to the pdf, my impression is that it is mainly just used by staff.

File: nominations/views.py (hunk 2 above)
Comment (ericbroder) [reply nominations/views.py]: I find this a little confusing but glad it looks right to you.

**Pull request merged**